src/protocols/aeternity/libs/AeternityAdapter.ts

```diff
@@ -194,8 +198,15 @@ export class AeternityAdapter extends BaseProtocolAdapter {
     // TODO
   }
 
-  async fetchTransactions(): Promise<any> {
-    // TODO
+  async fetchTransactions(
+    address:string,
+    nextPageParams:string | null,
+  ): Promise<IFetchTransactionResult> {
+    return fetchAeTransactions(
+      TXS_PER_PAGE,
+      address as Encoded.AccountAddress,
+      nextPageParams,
+    );
```

Comment:
Why not put all the logic of the `fetchAeTransactions` inside of the method itself?